src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -122,6 +122,8 @@ export class LargeArtistHeader extends Component<Props> {
             <Spacer my={2} />
           </Fragment>
         )}
+        <Spacer my={2} />
+        <Spacer my={2} />
```

Comment:
Why both `<Spacers>`?

Reply:
For some reason after rerender the single spacer was collapsing on itself and not rendering any margin. Having two fixes the issue and their margins collapse together so there's always 20px.

Reply:
Def a weird code smell -- maybe something like `<Box m={2} />` or `<Box m={2}>{"}</Box>`?

Reply:
Though it is weird that this is happening (haven't seen this before)